tokio/src/lib.rs

```diff
@@ -205,9 +205,10 @@
 //! ```
 //!
 //! If your code is CPU-bound and you wish to limit the number of threads used
-//! to run it, you should run it on another thread pool such as [rayon]. You
-//! can use an [`oneshot`] channel to send the result back to Tokio when the
-//! rayon task finishes.
+//! to run it, you should use a separate thread pool such as provided by [rayon] or
+//! a different [`Runtime`] than the one used to handle IO tasks.
+//! If using rayon, you can use an [`oneshot`] channel to send the result back
+//! to Tokio when the rayon task finishes.
```

Comment:
I am fine with making this change, but I think the phrasing here is not explicit enough. Maybe give a few more details on when exactly you can do this?

Reply:
I am sorry I am not clear on your suggestion -- what do you mean "when exactly you can do this"? Do you mean

1.  "when use a `oneshot` channel"?
2. "when you an use a different `Runtime`?"
3. Something else?

Reply:
I just want it to be more clear that the runtime should be dedicated to only CPU-bound tasks. It does sorta say that here, but I'd like it to be more precise about this.

Reply:
I attempted to clarify in f061eb22. 

I have also been thinking of drafting a blog post on this topic, which perhaps help clarify the options in a longer format than available in the docstrings.

Reply:
For anyone who cares, said post is now available at https://thenewstack.io/using-rustlangs-async-tokio-runtime-for-cpu-bound-tasks/